docs/examples/cas9_kymotracking/cas9_kymotracking.rst

```diff
@@ -67,6 +67,13 @@ of the colormap. Any photon count higher than that will be clipped to the maxima
 What we can observe in this data is that as more force is applied, we get an increased binding activity. Let’s see
 if we can put the kymotracker to some good use and quantify these.
 
+Compute background
+------------------
+First, we select a small region without traces to determine the background signal::
+
+    background=kymo['100s':'200s'].crop_by_distance(28,31)
+    green_background_ppixel=np.mean(background.green_image)
```

Comment:
Is is really `ppixel`, or just `pixel`? Also, the document apparently uses active voice for section titles. So `Compute the background` perhaps?

Reply:
I really meant to say 'per pixel'. And yes, it makes sense to use the same voice as in other section titles.

Reply:
Maybe `per_pixel` then?